### PR TITLE
feat: add annotations fetching->redux

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Connects #620 

Fetches annotations and stores them in redux. Doesn't render them yet.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: src/shared/components/RefreshingView.tsx [hunk 1]
```diff
@@ -62,7 +64,7 @@ class RefreshingView extends PureComponent<Props, State> {
   }
 
   public render() {
-    const {id, ranges, properties, manualRefresh, timeZone, theme} = this.props
+    const {id, manualRefresh, properties, ranges, theme, timeZone} = this.props
```
Comment (hoorayimhelping) [inline src/shared/components/RefreshingView.tsx:67]: alphabetization :shrug:

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 44 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: src/types/annotation.ts [hunk 2]
```diff
@@ -25,3 +25,5 @@ export interface DeleteAnnotation {
   end: string
   stickers?: any
 }
+
+export type StateAnnotations = {[stream: string]: Annotation[]}
```
Comment (hoorayimhelping) [inline src/types/annotation.ts:29]: I'm really not in love with this name. Up for suggestions

Comment (jrenee42) [reply src/types/annotation.ts:29]: AnnotationList

that is what it is.  
plus the AnnotationState / StateAnnotations in the reducers is confusing.

**Review by jrenee42 — COMMENTED** [review]
added suggestion:  AnnotationList instead of StateAnnotation for type

File: src/types/annotation.ts (hunk 2 above)
Comment (hoorayimhelping) [reply src/types/annotation.ts:29]: it's not quite a `list` though - it's a hashmap keyed on stream. It's kind of like a list, but kind of not

**Review by jrenee42 — APPROVED** [review]

**Pull request merged**